### ISSUE TITLE
Test snooping

### ISSUE THREAD

**Issue by carlthome** [sigsep/sigsep-mus-2018#4]
Is there a secret test set somewhere (like Kaggle and MIREX) that will be used for the evaluation? How do we know that submissions weren't accidentally tweaked for the public test set (e.g. cross validation and hyperparameter tuning)?

**Comment by aliutkus** [comment]
hi !

as usual for SiSEC, there is no hiddent test set: the whole test set is known to the participants, and the philosophy is to trust the participants for learning their systems on the train set, and to only use the test set for the final testing =)

**Comment by faroit** [comment]
As this sisec seems to be a great success, we can definitely think of another task for next year that includes a hidden test set that we compute on a private test server. But this would require a community effort.

Feel free to contact us at any time. I will close this out for now.
